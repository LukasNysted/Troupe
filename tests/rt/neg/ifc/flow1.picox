let
   val h = true raisedTo `{secret}`
   fun f () = adv 1
   fun g () = adv 2
in
   if h then spawn(f) else spawn(g) 
end
