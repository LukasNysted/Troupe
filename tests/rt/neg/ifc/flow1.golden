2023-03-17T21:23:23.789Z [RTM] [32minfo[39m: Skipping network creation. Observe that all external IO operations will yield a runtime error.
>>> Main thread finished with value: c89a969d-7152-472f-9a21-1365ce7b2143@{secret}%{secret}
Runtime error in thread c89a969d-7152-472f-9a21-1365ce7b2143@{secret}%{secret}
>> Illegal flow in adv function:
 |    pc: {secret}
 | block: {secret}
 | value: 1@{secret}%{secret}
